docs/operations/updating_the_stack.rst

```diff
@@ -34,6 +34,8 @@ Prerequisites
 =================================================================================
 This will ensure proper labels are applied to the docker swarm nodes.
 
+**It is recommended to install docker-auto-labels, deploy to staging, and promote to production in a virtual enviornment.**
```

Comment:
What kind of virtual environment? Just a python virtualenv because of docker-image-labels, or something heavier, like a Docker container or a VM?